packages/core/test/endowmentsToolkit.spec.js

```diff
@@ -457,3 +457,83 @@ test('copyWrappedGlobals - copy from prototype too', (t) => {
 
   t.is(Object.keys(target).sort().join(), 'onTheObj,onTheProto,window')
 })
+
+test('getEndowmentsForConfig - allow redefine on properties', (t) => {
```

Comment:
would prefer that we reduce the count of assertions in this test.  e.g. instead of 

```js
t.is(derivedGlobal.x, 1)
t.is(derivedGlobal.y, 2)
t.is(derivedGlobal.z, 3)
```

use

```js
t.like(derivedGlobal, {x: 1, y: 2, z: 3})
```

but splitting into multiple tests would be even better.

also use `t.plan()`

Reply:
lots of assertions is kinda my style, but I have no doubt you're right.